libecole/include/ecole/observation/strongbranchingscores.hpp

```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+#include <memory>
+
+#include <nonstd/optional.hpp>
+#include <xtensor/xtensor.hpp>
+#include <xtensor/xview.hpp>
+
+#include "ecole/environment/state.hpp"
+#include "ecole/observation/abstract.hpp"
+
+namespace ecole {
+namespace observation {
+
+class StrongBranchingScoresObs {
+public:
+	xt::xtensor<double, 1> scores;
+};
```

Comment:
Given that you have a single attribute, you can use that directly as the observation, that is
```cpp
class StrongBranchingScores : public ObservationFunction<nonstd::optional<xt::xtensor<double, 1>>> {
```
If that feels a bit verbose, you can also add an alias `template <typename T> using xvector = xt::xtensor<T, 1>;`